packages/chains/chains/314.ts

```diff
@@ -10,7 +10,7 @@ export default {
   "rpc": [
     "https://filecoin.rpc.thirdweb.com/${THIRDWEB_API_KEY}",
     "https://api.node.glif.io/",
-    "https://rpc.ankr.com/filecoin"
+    "https://rpc.ankr.com/filecoin, https://filecoin-mainnet.chainstacklabs.com/rpc/v1"
```

Comment:
this seems f'd but it's coming from source I guess?

Reply:
submitted a PR to fix this @ source: https://github.com/ethereum-lists/chains/pull/2431

Reply:
Lol good catch